### ISSUE TITLE
[Bug]  404s, bad host, timeouts, bad urls for URLs linked from website

### ISSUE THREAD

**Issue by Roshanjossey** [a11yproject/a11yproject.com#1478]
## Bug description

I checked links in the website with [brokenlinkcheck.com](https://www.brokenlinkcheck.com/) and found the following links could potentially have problems

| # | URL | link text | response |
|---|---|---|---:|
| 1 | https://jenniferbrownconsulting.com/inclusion-the-book/ | Inclusion: Diversity<COMMA> The New Workplace &amp; The Will | 404 |
| 2	 | https://www.getstark.co/newsletter | Stark | 404 |
| 3	 | https://www.a11yproject.com/posts/everyday-accessibility/A11yProject.com/Resources | The A11Y Project’s Resources Page | 404 |
| 4	 | https://chrome.google.com/webstore/detail/i-want-to-see-like-the-co/jebeedfnielkcjlcokhiobodkjjpbjia | Landmarks | 404 |
| 5	 | https://chrome.google.com/webstore/detail/nocoffee/jjeeggmbnhckmgdhmgdckeigabjfbddl | NoCoffee | 404 |
| 6	 | https://confrere.com/a11y/test/ | Accessibility Tester | 404 |
| 7	 | http://blindspot.fas.harvard.edu/Book | Blindspot: Hidden Biases of Good People | bad host |
| 8	 | http://colorfilter.wickline.org/ | Colorblind Web Page Filter | bad host |
| 9	 | https://hexnaw.com/ | Hex Naw | timeout |
| 10	 | https://image-color.com/color-contrast.html | Image-Color: Color Contrast Checker | 404 |
| 11	 | http://www.karlgroves.com/blog/ | Blog - Karl Groves | 404 |
| 12	 | https://pragmaticaccess.com/articles/2019-05-30-pdf-accessibility-introduction | PDF accessibility introduction | bad host |
| 13	 | https://www.funka.com/en/we-offer/funka-accessibility-days/ | Funka Accessibility Days | 404 |
| 14	 | https://www.meetup.com/a11yphl/ | A11yPHL - Philly Accessibility Meetup | 404 |
| 15	 | https://www.meetup.com/Munich-Accessibility-Meetup/ | Munich Accessibility Meetup | 404 |
| 16	 | https://www.meetup.com/Paris-Web-Accessibility/ | Paris Web Accessibility | 404 |
| 17	 | https://developer.yoast.com/accessibility-slow-down-development-process/ | The a11y Monthly: Does Accessibility slow down the deve | 404 |
| 18	 | https://usabilitygeek.com/information-different-types-people-graphic-communication-website-information-designers/ | Information On Different Types Of People For Graphic Co | 404 |
| 19	 | https://addons.mozilla.org/en-US/firefox/addon/visbug/ | VisBug | 404 |
| 20	 | https://blindjournalist.wordpress.com/2021/03/04/why-i-blocked-accessibe/ | Why I blocked #AccessiBe | 404 |
| 21	 | https://www.barrierbreak.com/2021/07/13/why-should-you-stay-away-from-accessibility-overlays/ | Why should you stay away from Accessibility Overlays? | 404 |
| 22	 | hhttps://www.w3.org/WAI/WCAG21/Understanding/understanding-techniques | Techniques | bad url |
| 23	 | http://ffb.ca/learn/eye-diseases/ | Retinal degeneration | 404 |
| 24	 | https://developer.mozilla.org/en-US/docs/Mozilla/Preferences/Preference_reference/accessibility.tabfocus | accessibility.tabfocus | 404 |
| 25	 | https://www.ncdt.nl/ncdt-2019/readability-on-the-web-damien-senger | research on readability on the web | 404 |
| 26	 | https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-invalid_attribute | aria-invalid=&quot;true&quot; | 404 |
| 27	 | https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-describedby_attribute | aria-describedby | 404 |
| 28	 | https://help.ubuntu.com/lts/installation-guide/s390x/ch01s01.html | Ubuntu | 404 |
| 29	 | https://www.pixel-ruler.net/android-scale-independent-pixel | scale-independent pixels | timeout |
| 30	 | https://www.mobileux.net/2019/06/24/designing-apps-with-accessible-fonts/ | Designing Apps with Accessible Fonts | 404 |
| 31	 | https://midnight-lizard.org/home | Midnight Lizard | 404 |
| 32	 | https://jamstackconf.com/jammies/ | Jammy for Social Impact Award | 404 |
| 33	 | http://jxnblk.com/colorable/demos/text/ | Colorable | 404 |

## Expected outcome

Not many 404s

## Steps to reproduce

Check if these links are working, use an automated tool to check

## Terms

- [x] I have read, and agree to follow this project's [Code of Conduct](https://www.a11yproject.com/code-of-conduct/)

**Comment by ericwbailey** [comment]
Thanks for submitting this, looks like we have some URLs to fix 😅

**Comment by p2635** [comment]
Potentially implement something like https://github.com/marketplace/actions/broken-link-checker-action to improve the process? Shall I create this as a new issue?

**Comment by ericwbailey** [comment]
@p2635 That'd be great, yes please!

**Comment by harsxv** [comment]
Actually, the [broken-link-checker-action](https://github.com/technote-space/broken-link-checker-action) is also "broken" since the owner archived the repository to read-only 😁

**Comment by p2635** [comment]
Thanks for pointing that out, that was just an example @harsxv. I was just linking that as an idea. I've seen other link checkers and they are not perfect, but hopefully they can still save some time.

**Comment by meatnordrink** [comment]
Yeah, the Newsletter link, which was just posted on Mastodon, is totally broken. 

(Tried to tell you on Mastodon, but you're not allowing new accounts now, nor allowing accounts created elsewhere to sign in on your servers...?)

**Comment by davatron5000** [comment]
@meatnordrink Apologies. Mastodon ownership handoff is still in-progress. Will be awhile before that gets sorted out and a few mistakes might go through the auto-poster.

**Comment by ericwbailey** [comment]
To clarify, `@a11y.info` is not owned by The A11Y Project organization. The account is hosted on the service, which is maintained by another individual. The A11Y Project are not allowing or disallowing anything in this regard.

The newsletter link was shared over two weeks ago, [as per its post date](https://a11y.info/@thea11yproject/110357073171353645). The page itself has been intentionally removed, and is not "totally broken."

**Comment by westonludeke** [comment]
Hi A11Y Project team! 

I'm Weston👋 I love the work y'all are doing!

I saw this issue is still open and is labeled as a 'good-first-issue'. I've never committed a pull request to an open source project before, so I thought this would be a good chance for me to dip my toes in the water.

I updated three links and submitted a pull request: https://github.com/a11yproject/a11yproject.com/pull/1584

Let me know if there's anything I need to fix. If all goes well, I'm happy to continue to review other links.

Cheers!

**Comment by p2635** [comment]
Is Roshan still around to check off whatever is done on the main list of links? It's hard for me to know what is remaining.

**Comment by p2635** [comment]
I checked it against my automated github action which I am investigating on https://github.com/a11yproject/a11yproject.com/issues/1521#issuecomment-1933607034

These appear to be the outstanding broken links:

- https://blindjournalist.wordpress.com/2021/03/04/why-i-blocked-accessibe/
- https://www.barrierbreak.com/2021/07/13/why-should-you-stay-away-from-accessibility-overlays/
- https://developer.mozilla.org/en-US/docs/Mozilla/Preferences/Preference_reference/accessibility.tabfocus
- https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-invalid_attribute
- https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-describedby_attribute
- https://help.ubuntu.com/lts/installation-guide/s390x/ch01s01.html
- https://www.pixel-ruler.net/android-scale-independent-pixel
- https://midnight-lizard.org/home (this appears to be a false positive, I can visit the site fine)
- http://jxnblk.com/colorable/demos/text/

**Comment by evelinabe** [comment]
I just submitted a new pull request,  #1596. It updates or removes some of the broken links above:

- https://www.funka.com/en/we-offer/funka-accessibility-days/ removed section
- https://yoast.com/developer-blog/accessibility-slow-down-development-process/ updated
- https://medium.com/@user_design/information-on-different-types-of-people-for-graphic-communication-website-and-information-4b6454905bb1 updated with info about new destination
- https://blindjournalist.wordpress.com/2021/03/04/why-i-blocked-accessibe/ removed
- http://ffb.ca/learn/eye-diseases/ changed link to https://www.sciencedirect.com/topics/neuroscience/retinal-degeneration 
- http://colorfilter.wickline.org/ is removed already
- https://hexnaw.com/ is working
- https://addons.mozilla.org/en-US/firefox/addon/visbug/ is working
